src/Sentry/SpanId.cs

```diff
@@ -41,7 +41,7 @@ namespace Sentry;
     public override int GetHashCode() => StringComparer.Ordinal.GetHashCode(_value);
 
     /// <inheritdoc />
-    public override string ToString() => _value.ToString("x8");
+    public override string ToString() => _value.ToString("x8").PadLeft(16, '0');
```

Comment:
Is it possible to create a unit test that reproduces the issue (prior to this fix) and then apply this fix to make the test pass? That would prevent any regressions in future.

Reply:
Done!